keywording/text.xml

```diff
@@ -232,13 +232,6 @@ new bug</uri> to the architecture in question regarding this if no bug is
 already available.
 </p>
 
-<p>
-If there are no new dependencies, do not remove keywords if your commit fails
-with <c>repoman</c> or <c>pkgdev</c> <d/> instead please commit with
-<c>repoman -I</c> and file a bug to the broken architecture, noting it in your
-git commit message.
-</p>
-
```

Comment:
I discussed this with @floppym and @benkohler on `#gentoo-dev`. As best as we can figure, this is referring to a situation where the depgraph is *already* broken, and you're just trying to make a version bump.

The text doesn't make a ton of sense because it's telling you to commit with `repoman -I` but `repoman(1)` describes `-I, --ignore-masked` as "Ignore masked packages (not allowed with commit mode)".

So we figure the best thing to do is to simply drop the paragraph.